typo

Karmavil opened the pull request:
replaced it for if. Line 230 ("to be given if this flow is being used")

LilSpazJoekp reviewed (approved):
Hey, thanks for catching that! Would you mind adding yourself to the Documentation Contributors section in the AUTHORS.rst file?

Karmavil commented:
I know I have to do a lot more to put my name there, but you have to start somewhere, right?

I'm sorry for the delay.  I thought I could do the edit from the browser here on github but it created a new branch for me. I already removed it and added the commit to this branch.

LilSpazJoekp commented:
Thank you! 🎉
